### PR TITLE
Fixed Spy Survived Event missing Icon and always triggering

### PR DESCRIPTION
**Pull request by ZacharyHinds**
Added a resource.AddFile for the icon to ensure it's loaded on client.

Added a check before triggering the event to see if there are actually any Spies currently alive.

### CODE REVIEW

**Comment by TimGoll** [comment]
Oh thanks. I already had this on my todo list for tomorrow!

File: lua/terrortown/events/spy_alive.lua [hunk 1]
```diff
@@ -1,5 +1,10 @@
 EVENT.base = "base_event"
 
+if SERVER then
+	AddCSLuaFile()
```
Comment (TimGoll) [inline lua/terrortown/events/spy_alive.lua]: Not needed here

Comment (ZacharyHinds) [reply lua/terrortown/events/spy_alive.lua]: Removed

**Comment by TimGoll** [comment]
Okay, I looked at your code and it would add mutiple events for multiple spies alive. So I changed that. And while doing that I realised that I could just put it in the already existing loop. I'd appreciate it if you could verify that it still works as you intended it. But it works for me.

**Comment by ZacharyHinds** [comment]
Working as intended for me

**Comment by TimGoll** [comment]
Thanks as always!

**Pull request merged**